experimental/parser/testdata/parser/method/incomplete.proto.stderr.txt

```diff
@@ -5,16 +5,16 @@ error: missing `(...)` around method return type
    |                               ^^^^^^^ help: replace this with `(foo.Bar)`
 
 error: missing method return type in service method
-  --> testdata/parser/method/incomplete.proto:21:5
+  --> testdata/parser/method/incomplete.proto:21:13
    |
 21 |     rpc Bar2(foo.Bar);
-   |     ^^^^^^^^^^^^^^^^^^
+   |             ^^^^^^^^^ expected type in `(...)` after this
```

Comment:
Seems like we really want to know there was no "returns" keyword and make the message a little more clear. LIke complain that we want a returns clause here, instead of just the type in parentheses.

Reply:
Added a new taxon for this.